tests/unit/dashboard/route-test.js

```diff
@@ -0,0 +1,11 @@
+import { moduleFor, test } from 'ember-qunit';
+
+moduleFor('route:dashboard', 'Unit | Route | dashboard', {
+    // Specify the other units that are required for this test.
+    // needs: ['controller:foo']
```

Comment:
Every route test needs` ['service:metrics']` now that we have page tracking in the router.